### ISSUE TITLE
New major release v7

### ISSUE THREAD

**Issue by lucdekens** [alanrenouf/vCheck-vSphere#700]
A new major release (v7) should be targetted.
This should include:

- closing open issues
  - bug fixes
  - feature requests
- consolidate all pending PRs
- ???

**Comment by haripetrov** [comment]
Who will work on this new release?

**Comment by lucdekens** [comment]
I intend to spend some time on that.

**Comment by Outek** [comment]
Just a Question, what if we try to move all the checks to Pester Tests and build a Module out of it?

**Comment by lucdekens** [comment]
We could then rename vCheck to Vester ;-)

Seriously though, there is already an open source project that does exactly that, and more.
See https://github.com/WahlNetwork/Vester

**Comment by Outek** [comment]
Yeah, right. I already contributed 1-2 simple Tests to Vester. I think, vCheck should look or feel a bit more... modern or how you like to call it. Something like a Container. Run it with a few Variables and right after the checks it displays the results as a static webpage. Maybe with a Universal Dashboard?

**Comment by lucdekens** [comment]
All good ideas.
We plan on presenting a kind of roadmap soon-ish (probably early 2020), with some mock-ups.

A Dashboard might be nice, on the other hand, we don't want to make vCheck too dependent on other products/modules.
We could consider an additional output format that could be consumed by other products, like Universal Dashboard.

Keep those ideas/suggestions coming!

**Comment by monahancj** [comment]
A dashboard feel like feature creep to me.  A more modular, "PowerShelly", approach might be to have an option to dump all the data in or more CSV(s) or other dashboard friendly file format that your dashboard of choice can consume.

**Comment by paulharman** [comment]
Is this project discontinued now?  Anyone know of any alternatives?

**Comment by lucdekens** [comment]
No, it is not.
It is delayed because ... 2020 :-(

**Comment by paulharman** [comment]
That's fair xD
